static/js/src/advantage/react/components/Subscriptions/SubscriptionList/SubscriptionList.tsx

```diff
@@ -86,6 +87,21 @@ const SubscriptionList = ({ selectedId, onSetActive }: Props) => {
     )
   );
 
+  const checkActiveSubscription = (subscriptions: UserSubscription[]) => {
+    let activeSubscription = false;
+    subscriptions?.length > 0 &&
+      subscriptions?.forEach((subscription) => {
+        if (subscription?.statuses?.is_subscription_active == true) {
+          activeSubscription = true;
```

Comment:
Since activeSubscription is only instantiated as false at the start, can the loop be escaped early if an activeSubscription is found?

Reply:
Thanks Britney! Yep that sounds better! I will use `some()` method , which immediately returns `true` and stops iterating through the array.